Makefile

```diff
@@ -63,7 +64,10 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+
+yarn.lock: package.json
+	$(YARN)
```

Comment:
Is `yarn` short for `yarn install`? If so let's use `$(YARN) install` here for clarity.